app/views/contexts/summary/question.py

```diff
@@ -174,12 +192,19 @@ def _get_dynamic_answer_options(
 
         return dynamic_options.evaluate()
 
-    def get_answer_options(self, answer_schema):
+    def get_answer_options(
+        self, answer_schema: Mapping[str, Any]
+    ) -> tuple[dict[str, str], ...]:
         return tuple(answer_schema.get("options", ())) + tuple(
             self._get_dynamic_answer_options(answer_schema)
         )
 
-    def _build_checkbox_answers(self, answer, answer_schema, answer_store):
+    def _build_checkbox_answers(
+        self,
+        answer: Markup,
```

Comment:
How come this is markup?

Reply:
As the flow is coming from the template its the string ready to be safely inserted into a HTML. This can be changed to str. Will modify

Reply:
👍